etc/configure.py

```diff
@@ -0,0 +1,29 @@
+import subprocess
+import os
+import virtualenv
+
+req_path = os.path.abspath(os.path.join(os.pardir, 'scancode-server'))
+
+
+def install_requirements():
+    commands = ['pip', 'install', '-r', req_path + '/requirements.txt']
+    subprocess.call(commands)
+
+
+def create_venv():
+    venv_dir = os.path.join(req_path, ".scancode-server")
```

Comment:
Can we be consistent with the quotations ? Either `"`(double quotes) everywhere or `'`(single quotes) everywhere ?

Reply:
we use single quote everywhere unless this is a doc string (or if the string contains single quotes)